src/components/SlideButton/SlideButton.stories.js

```diff
@@ -0,0 +1,45 @@
+import { action } from '@storybook/addon-actions';
+import { withKnobs, select } from '@storybook/addon-knobs';
+
+import StorybookMobileDeviceSimulator from '../StorybookMobileDeviceSimulator/StorybookMobileDeviceSimulator.vue';
+import { availableDevices } from '../StorybookMobileDeviceSimulator/StorybookMobileDeviceSimulator';
+import SlideButton from './SlideButton.vue';
+
+const SlideButtonStories = {
+  component: SlideButton,
+  title: 'Components/SlideButton',
+  decorators: [withKnobs],
+};
+
+const defaultExample = () => ({
+  components: {
+    SlideButton,
+    StorybookMobileDeviceSimulator,
+  },
+  props: {
```

Comment:
Please add two more knob props for allowing Storybook users to edit the `label` and `completedLabel` prop values, and then bind them to the `<SlideButton>` element on line: 32, below.

Reply:
Please address this comment and let me know if you'd like any help ^